src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -48,6 +48,9 @@ namespace onnx_mlir {
 
 void configurePassesNNPA() {
   configureOnnxToZHighLoweringPass(optReport == OptReport::NNPAUnsupportedOps);
+  // Compiler generated sticks supports saturation, so force its usage.
+  if (nnpaEnableSaturation)
+    nnpaEnableCompilerStickUnstick = true;
```

Comment:
Since `--nnpa-enable-compiler-stick-unstick` is enabled by default, it seems we don't need to force this so that we can have "nnpa-saturation=true --nnpa-enable-compiler-stick-unstick=false" if NNPA supports stick/unstick.

Reply:
> Does it make sense to add a "TODO, remove this if zDNN supports saturation"

That sounds good if you agree.

Reply:
of course.